version/version_lib.go

```diff
@@ -0,0 +1,16 @@
+// +build !autogen
+
+// Package version is auto-generated at build-time
+package version
+
+// Default build-time variable for library-import.
+// This file is overridden on build with build-time informations.
+const (
+	GITCOMMIT string = "library-import"
```

Comment:
@stevvooe I guess this is based https://github.com/docker/docker/pull/17350#discussion_r43043783 and the fact that the generated file were already like that (so didn't changed it). And `golint` is actually *OK* with ALL_CAPS constants.